NO_PUB_FEATURE.md

```diff
@@ -0,0 +1,130 @@
+# --no-pub Feature Implementation
```

Comment:
This should not be its own file, it should follow the standard for the rest of the documentation.